src/MSALWrapper.Benchmark/MSALWrapper.Benchmark.csproj

```diff
@@ -1,46 +1,37 @@
-        <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
-    </PropertyGroup>
-
-    <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('OSX'))">
-        <TargetFramework>net6.0</TargetFramework>
-    </PropertyGroup>
-
-    <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('Linux'))">
-        <TargetFramework>net6.0</TargetFramework>
-    </PropertyGroup>
-
-    <PropertyGroup>
-        <ImplicitUsings>enable</ImplicitUsings>
-        <Nullable>enable</Nullable>
-        <RootNamespace>Microsoft.Authentication.$(MSBuildProjectName.Replace(" ", "_"))</RootNamespace>
-        <OutputType>Exe</OutputType>
-    </PropertyGroup>
-    <ItemGroup>
-        <PackageReference Include="BenchmarkDotNet" Version="0.13.2" />
-        <PackageReference Include="Microsoft.Identity.Client.Broker" Version="4.61.3" />
-    </ItemGroup>
-    <ItemGroup>
-        <ProjectReference Include="..\MSALWrapper\MSALWrapper.csproj" />
-    </ItemGroup>
-
-    <ItemGroup Condition="$([MSBuild]::IsOSPlatform('Windows'))">
-        <!--Windows: Explicitly continue using the net5 version of MSAL since there is no net6 target yet 
+  <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('Windows'))">
+    <TargetFramework>net8.0</TargetFramework>
+    <DefineConstants>PlatformWindows</DefineConstants>
+    <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
+  </PropertyGroup>
+  <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('OSX'))">
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+  <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('Linux'))">
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+  <PropertyGroup>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <RootNamespace>Microsoft.Authentication.$(MSBuildProjectName.Replace(" ", "_"))</RootNamespace>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.2" />
+    <PackageReference Include="Microsoft.Identity.Client.Broker" Version="4.61.3" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\MSALWrapper\MSALWrapper.csproj" />
+  </ItemGroup>
+  <ItemGroup Condition="$([MSBuild]::IsOSPlatform('Windows'))">
+    <!--Windows: Explicitly continue using the net5 version of MSAL since there is no net6 target yet 
          https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3682-->
-        <PackageReference Include="Microsoft.Identity.Client" GeneratePathProperty="true">
-            <Version>4.61.3</Version>
-        </PackageReference>
-
-        <Reference Include="Microsoft.Identity.Client">
-            <HintPath>$(PkgPMicrosoft_Identity_Client)\lib\net5.0-windows10.0.17763\Microsoft.Identity.Client.dll</HintPath>
-        </Reference>
-    </ItemGroup>
-    
-    <ItemGroup Condition="$([MSBuild]::IsOSPlatform('Windows')) == false">
+    <Reference Include="Microsoft.Identity.Client">

```

Comment:
Is the `GeneratePathProperty="true"` that was here just not needed for some reason?

Reply:
oh good catch!  i remember that whole block was removed by .NET Upgrade Assistant initially. 
```
  <PackageReference Include="Microsoft.Identity.Client" GeneratePathProperty="true">
            <Version>4.61.3</Version>
  </PackageReference>
```


I must have accidentally deleted that field which trying to add it back